regolith/helpers/a_manurevhelper.py

```diff
@@ -0,0 +1,122 @@
+"""Builder for manuscript reviews."""
+import datetime as dt
+import sys
+import time
+from argparse import RawTextHelpFormatter
+
+import nameparser
+
+from regolith.helpers.basehelper import SoutHelperBase, DbHelperBase
+from regolith.dates import month_to_int, month_to_str_int
+from regolith.fsclient import _id_key
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    filter_grants,
+    fuzzy_retrieval,
+)
+
+ALLOWED_STATI = ["invited", "accepted", "declined", "downloaded", "inprogress",
+                 "submitted", "cancelled"]
+
+
+def subparser(subpi):
+    subpi.add_argument("name", help="Full name of the first author",
+                       )
+    subpi.add_argument("due_date", help="due date in form YYYY-MM-DD in quotes", default=''
+                       )
+    subpi.add_argument("-d", "--database",
```

Comment:
move this to the bottom of the list.  It is rarely used.  Also, remove the ``-d`` (as it is rarely used)

Reply:
Completed.